package.json

```diff
@@ -235,13 +235,15 @@
     "is-elevated": "^3.0.0",
     "is-root": "^2.1.0",
     "js-yaml": "^3.14.0",
+    "JSONStream": "^1.0.3",
     "klaw": "^3.0.0",
     "livepush": "^3.5.0",
     "lodash": "^4.17.20",
     "minimatch": "^3.0.4",
     "mixpanel": "^0.10.3",
     "moment": "^2.27.0",
     "moment-duration-format": "^2.3.2",
+    "net-keepalive": "^1.3.6",
```

Comment:
I think `net-keepalive` and `windosu` were listed only in the `optionalDependencies` section, not the `dependencies` section, because they are not available for all platforms:
* `net-keepalive` require a native node module compilation that only succeeds on Linux and macOS, not Windows, and causes the build and installation to fail on Windows. (The CLI dynamically checks the platform before loading it at runtime.)
* `windosu` is only needed on Windows (iirc, the CLI won't try to use it on other platforms).

Were these changes necessary for any specific reason? If not, I suggest reverting them. The `net-keepalive` change is indeed currently preventing the build from succeeding on Windows.

Reply:
Yeah npm did that for me, how nice.

Reply:
It looks like you're using `npm` v7 again, and committed these changes again, alongside a huge `npm-shrinkwrap.json` file that will break the build again. Please use `npm` v6.